packages/dashboard/src/components/common/Popover.tsx

```diff
@@ -0,0 +1,22 @@
+import { Popover } from '@headlessui/react';
+import SettingsIcon from "./SettingsIcon";
+import AnalyticsToggle from "./AnalyticsToggle";
+
+export default function SettingsPopover() {
+  return (
+    <Popover>
+      <Popover.Button className="absolute bottom-2 right-2"> <SettingsIcon analyticsSet={false} /></Popover.Button >
+      <Popover.Overlay className="bg-black opacity-30 fixed inset-0" />
+      <Popover.Panel className="absolute z-10 bottom-14 right-2 mb-2">
+        <div className="bg-truffle-light border-2 mb-4 block z-50 font-normal leading-normal text-lg max-w-xs no-underline break-words rounded-lg">
+          <div className="bg-truffle-blue border-2 text-center text-truffle-brown font-semibold p-3 mb-2 uppercase rounded-t-lg">Help Improve Truffle!</div>
+          <div className="bg-truffle-light text-truffle-brown m-2">Truffle uses anonymous opt-in analytics to determine how we could make our
+            product better. Are you willing to opt-in?
```

Comment:
The question "Are you willing to opt-in?" cannot be answered by the words "Enabled" or "Disabled".

Also, if the options were "Yes" or "No" to this question a "Yes" answer would not actually give us permission to track them. It only lets us know that if we did ask for permission they would be willing to opt in.

I propose something like:

> You can help us improve Truffle by allowing us to track how you use our tools. For details on what information we collect and how we use it visit our [tracking policy].
> Would you like to allow tracking?
> [✔️] Allow Tracking

Reply:
Do we have a tracking policy link? I wasn't aware of one. We had discussed the format and had decided on a slider. I see what you mean about disabled/enabled and will change it to yes/no. I like your rewording, only open question being whether we actually have a tracking policy!

Reply:
Oh also, yes there is a conditional for the red dot, hardcoded to include it for now. Liang is taking this over to add the backend component at which point the conditional will actually be hooked into something!

Reply:
I reworded this but didn't add anything about the tracking policy because I do not believe we have one yet. Definitely think we should add a link when we can though.